test/ldapsync.py

```diff
@@ -116,3 +117,7 @@ def test_sync_altered_groups(self, mock_add_or_update):
         ldap_attrs =  {'status': ['active'], 'cn': 'vsc40075', 'institute': ['gent'], 'memberUid': ['vsc40075'],
                        'moderator': ['vsc40075'], 'gidNumber': ['2540075']}
         mock_add_or_update.assert_called_with(VscLdapGroup, test_group.vsc_id, ldap_attrs, True)
+
+
+if __name__ == '__main__':
```

Comment:
what is this about?

Reply:
being able to run the test directly.

Reply:
you don't need this to do so; `python setup.py test --help` there's options to select test module and function

Reply:
That doesn't work as `vsc-ldap-extension` is not available (see #77). And it's faster.

Reply:
we should somehow fix the issue, but we are not going to pollute all unittests files with this.

Reply:
and also ` python -m unittest test.slurm` etc etc